lib/mvi/service.rb

```diff
@@ -51,10 +51,10 @@ def find_profile(user_identity)
       end
     rescue Breakers::OutageException => e
       Raven.extra_context(breakers_error_message: e.message)
-      log_console_and_sentry('MVI find_profile connection failed.', :error)
+      log_console_and_sentry('MVI find_profile connection failed.', :info)
       mvi_profile_exception_response_for('MVI_503', e)
     rescue Faraday::ConnectionFailed => e
-      log_console_and_sentry("MVI find_profile connection failed: #{e.message}", :error)
+      log_console_and_sentry("MVI find_profile connection failed: #{e.message}", :info)
       mvi_profile_exception_response_for('MVI_504', e)
     rescue Common::Client::Errors::ClientError, Common::Exceptions::GatewayTimeout => e
       log_console_and_sentry("MVI find_profile error: #{e.message}", :error)
```

Comment:
whatever is decided, this should be the same level as the others I would think.

Reply:
I tend to agree and had originally done a slight refactor to make `Common::Exceptions::GatewayTimeout` as `INFO` as well (but not `Common::Client::Errors::ClientError`). 

However, I then [searched the error string](http://sentry.vetsgov-internal/vets-gov/platform-api-production/?query=is%3Aunresolved+MVI+find_profile+error) and I'm not confident that I _wouldnt_ want to be alerted on at least some of the following:
![image](https://user-images.githubusercontent.com/3077884/61243129-4ed14480-a715-11e9-83e5-d23977010f1d.png)

I also can't figure out a sure-fire way of determining which of these error messages maps to which exception type other then using my gut.

Reply:
`:warn` makes more sense to me :+1: